weasyl/commishinfo.py

```diff
@@ -56,16 +60,48 @@ def parse_currency(target):
 
 
 @region.cache_on_arguments(expiration_time=60 * 60 * 24)
-def _fetch_rates():
+def _fetch_rates_xml():
     """
-    Retrieves most recent currency exchange rates from fixer.io, which uses
-    the European Central Bank as its upstream source.
+    Retrieve most recent currency exchange rates from the European Central Bank.
 
-    This value is cached with a 24h expiry period
-
-    :return: see http://fixer.io/
+    This value is cached with a 24h expiry period.
     """
-    return None
+    if not config.config_read_bool('convert_currency'):
```

Comment:
Is this written as intended? If I am understanding ``config.config_read_bool`` correctly, it should pull in the boolean name, then read the value inside of the value name. So as is currently in the example config:
```
+# Whether currency exchange rates should be fetched for /marketplace
+convert_currency = false
```
Would that evaluate to ``if not False:``, so ``True``, then? And if it were set to true, wouldn't this evaluate to False? (Assuming, of course, I am not misreading this.)